### PR TITLE
test: add test empty config is sufficient

### PR DESCRIPTION
**Pull request by mattsse**
add another test to ensure setting `defaultNetwork = "anvil"` is sufficient

### CODE REVIEW

File: packages/hardhat/test/fixture-projects/no-anvil-config/hardhat.config.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// We load the plugin here.
+import { HardhatUserConfig } from "hardhat/types";
+
+import "../../../src/index";
+
+const config: HardhatUserConfig = {
+  solidity: "0.7.3",
+  defaultNetwork: "anvil"
```
Comment (gakonst) [inline packages/hardhat/test/fixture-projects/no-anvil-config/hardhat.config.ts:8]: can we not require this at all? have the hardhat plugin do `hre.config.defaultNetwork = "anvil"` for the user?

**Review by gakonst — CHANGES_REQUESTED** [review]
tests seem to fail

**Comment by mattsse** [comment]
no idea why checks are not running...
